Polyhedron/demo/Polyhedron/MainWindow.cpp

```diff
@@ -1945,3 +1928,34 @@ void MainWindow::resetHeader()
 }
 
 
+void MainWindow::reset_default_loaders()
+{
+  default_plugin_selection.clear();
+
+  const char* prop_name = "Menu modified by MainWindow.";
+  QMenu* menu = ui->menuFile;
+  if(!menu)
+    return;
+  bool menuChanged = menu->property(prop_name).toBool();
+  if(!menuChanged) {
+    menu->setProperty(prop_name, true);
+  }
+  QList<QAction*> menuActions = menu->actions();
+  menu->removeAction(actionResetDefaultLoaders);
+}
+
+void MainWindow::insertActionBefore(QMenu* menu, QAction* actionToInsert)
```

Comment:
- Either the name should be changed to `'insertActionBeforeLoadPlugin`,
- or the function should take a third parameter: `insertActionBefore(QMenu* menu, QAction* action, QAction* actionToInsert)`.